public/javascripts/SVValidate/src/status/StatusField.js

```diff
@@ -46,9 +46,16 @@ function StatusField() {
         var prefix = createPrefix(labelType);
 
         // Centers and updates title top of the validation interface.
-        svv.ui.status.upperMenuTitle.html("Is this " + prefix + labelName.bold() + "?");
-        var offset = svv.ui.status.upperMenuTitle.width();
-        var width = (svv.canvasWidth - offset) / 2;
+        if (labelName === "No Sidewalk") {
+            svv.ui.status.upperMenuTitle.html("Should there be a " + "No Sidewalk".bold() + " label here?");
+        } else {
+            svv.ui.status.upperMenuTitle.html("Is this " + prefix + labelName.bold() + "?");
+        }
+        var offset = svv.ui.status.upperMenuTitle.width()
+            + $("#zoom-in-button").width()
```

Comment:
I'm storing all the jQuery objects in objects in `Main.js`. I'm not sure if these are best practices or not, but it's what I've been trying to be consistent with.

For zoom in (for example) could you do something like this in `Main.js`:
```
svv.ui.status.zoomInButton = $("#zoom-in-button");
```

and change line 55 in this file to:
```
+ svv.ui.status.zoomInButton.width()
```

Reply:
perfect, thanks, will do!

Reply:
done!